src/main/resources/mappings/opendistro-ism-history.json

```diff
@@ -93,6 +93,17 @@
                       "type": "keyword"
                     }
                   }
+                },
+                "rollup_id": {
+                  "type": "text",
+                  "fields": {
+                    "raw": {
```

Comment:
Let's change to "keyword", not sure when "raw" got introduced but we have a mix of "raw" and "keyword" now for the sub field type keyword and majority is keyword so we'll try to stick with that.

Reply:
sure, changed